src/components/transport-mode/transport-icon/transport-icon.module.css

```diff
@@ -6,15 +6,24 @@
   gap: 0.125rem;
 }
 
-.transportIcon {
+.transportIcon,
+.transportIconWithLabel {
   display: inline-block;
 
   /* Specialized sizing. */
   /* @TODO investigate generic way to handle this */
```

Comment:
Comment no longer true?
```suggestion
```

Reply:
Sort of. I've deviated from the design here I see, and used a variable. We should go through the design and the implementations and look at stuff like this. I think it's preferrable to use set variables.